doc/users/prev_whats_new/whats_new_3.3.0.rst

```diff
@@ -440,19 +635,16 @@ the documentation for `.FigureCanvasSVG.print_svg`.
 .. _RDF: https://www.w3.org/1999/.status/PR-rdf-syntax-19990105/status
 
 Saving PDF metadata via PGF now consistent with PDF backend
------------------------------------------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 When saving PDF files using the PGF backend, passed metadata will be
 interpreted in the same way as with the PDF backend.  Previously, this metadata
 was only accepted by the PGF backend when saving a multi-page PDF with
 `.backend_pgf.PdfPages`, but is now allowed when saving a single figure, as
 well.
 
-`.backend_bases.key_press_handler` and `.backend_bases.button_press_handler` simplifications
---------------------------------------------------------------------------------------------
+NbAgg and WebAgg no longer use jQuery & jQuery UI
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-These event handlers can now be directly connected to a canvas with
-``canvas.mpl_connect("key_press_event", key_press_handler)`` and
-``canvas.mpl_connect("button_press_event", button_press_handler)``, rather than
-having to write wrapper functions that fill in the (now optional) *canvas* and
-*toolbar* parameters.
+Instead, they are implemented using vanilla JavaScript. Please report any
```

Comment:
plain Javascript

Reply:
Vanilla JS is a ['thing'](http://vanilla-js.com/).

Reply:
yes, that's why I was kinda confused about this terminology here, didn't grok til just now that that's a joke page.